react/src/components/footer/SprkFooter.stories.js

```diff
@@ -201,14 +211,14 @@ const awards = {
     {
       href: '#nogo',
       src:
-        'https://spark-assets.netlify.com/spark-logo.svg',
+        'https://spark-assets.netlify.com/spark-logo-black-text.svg',
```

Comment:
Whats the image change?

Reply:
it makes it black and readable instead of white text